lib/src/generator/generator_frontend.dart

```diff
@@ -34,6 +35,21 @@ class GeneratorFrontEnd implements Generator {
   /// Traverses the package graph and generates documentation for all contained
   /// elements.
   List<Indexable> _generateDocs(PackageGraph packageGraph, FileWriter writer) {
+    runtimeStats.resetAccumulators([
```

Comment:
It would be slightly better to use constant `Set`, because then the compiler will complain if there are duplicates :-)

Reply:
Good point! Done